storm-client/src/jvm/org/apache/storm/security/auth/AutoSSL.java

```diff
@@ -100,19 +100,20 @@ public void populateCredentials(Map<String, String> credentials) {
     // the key.
     public static void serializeSSLFile(String readFile, Map<String, String> credentials) {
         try {
-            FileInputStream in = new FileInputStream(readFile);
-            LOG.debug("serializing ssl file: {}", readFile);
-            ByteArrayOutputStream result = new ByteArrayOutputStream();
-            byte[] buffer = new byte[4096];
-            int length;
-            while ((length = in.read(buffer)) != -1) {
-                result.write(buffer, 0, length);
-            }
-            String resultStr = DatatypeConverter.printBase64Binary(result.toByteArray());
+            try(FileInputStream in = new FileInputStream(readFile)) {
```

Comment:
space between `try` and `(`, and let's not make it unnecessary nested. It can be flattened.